recipe/download_osx_sdk.sh

```diff
@@ -29,6 +31,20 @@ if [[ ! -d ${CONDA_BUILD_SYSROOT} ]]; then
         url="https://github.com/phracker/MacOSX-SDKs/releases/download/11.3/MacOSX${MACOSX_SDK_VERSION}.sdk.tar.xz"
     fi
     curl -L --output MacOSX${MACOSX_SDK_VERSION}.sdk.tar.xz "${url}"
+    # As of 2024-04-02, hashes are coming from:
+    # - SDKs 12+ list a hash in the descriptions on https://github.com/alexey-lysiuk/macos-sdk/releases 
+    # - SDKs <12 were generated by running `openssl sha256` on artifacts at https://github.com/phracker/MacOSX-SDKs/releases/tag/11.3
+    sdk_sha256=$(
+        case "${MACOSX_SDK_VERSION}" in
+            ("13.3") echo "71ae3a78ab1be6c45cf52ce44cb29a3cc27ed312c9f7884ee88e303a862a1404" ;;
+            ("12.3") echo "91c03be5399be04d8f6b773da13045525e01298c1dfff273b4e1f1e904ee5484" ;;
+            ("11.3") echo "cd4f08a75577145b8f05245a2975f7c81401d75e9535dcffbb879ee1deefcbf4" ;;
+            ("11.1") echo "68797baaacb52f56f713400de306a58a7ca00b05c3dc6d58f0a8283bcac721f8" ;;
+            ("11.0") echo "d3feee3ef9c6016b526e1901013f264467bb927865a03422a9cb925991cc9783" ;;
+            ("10.9") echo "fcf88ce8ff0dd3248b97f4eb81c7909f2cc786725de277f4d05a2b935cc49de0" ;;
```

Comment:
we need the higher 10.x versions as we are going to update this soon.

Reply:
Oops, lexicographic sorting made me overlook 10.10-10.15 and somehow went from 10.9 to 11.0 straight. Sorry! Added https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/312